_posts/2020-02-20-whats-coming-in-eslint-7.0.0.md

```diff
@@ -58,16 +58,19 @@ Directive comment descriptions allow you to leave a note to yourself and other d
 
 You can read more in the [RFC](https://github.com/eslint/rfcs/pull/33).
 
-### Deprecating `~/.eslintrc`
+### Warnings for using `~/.eslintrc`
 
-Config files in your home directory are deprecated in v7.0.0. This feature has long been a source of confusion for ESLint users, especially because ESLint only looks for a config file in the home directory if no other config file is found. The configuration system in ESLint has changed a lot since home directory config files were introduced, and with the additional flexibility we feel there are plenty of other options for configuring ESLint that are clearer than the home directory config file.
+Config files in your home directory were deprecated in v6.7.0 and v7.0.0 will warn when one is used. This feature has long been a source of confusion for ESLint users, especially because ESLint only looks for a config file in the home directory if no other config file is found. The configuration system in ESLint has changed a lot since home directory config files were introduced, and with the additional flexibility we feel there are plenty of other options for configuring ESLint that are clearer than the home directory config file.
 
 Support for home directory config files will be removed in v8.0.0. You can read more in the [RFC](https://github.com/eslint/rfcs/pull/32).
 
 ### Updating the base path when using `--config` or `--ignore-path`
 
 In versions prior to v7.0.0, ESLint would resolve references inside of config files or ignore files relative to the file location passed to `--config` or `--ignore-path`. This didn't make much sense and was a source of confusion and unexpected behaviors for users. In v7.0.0, config files and ignore files passed using `--config` or `--ignore-path` use the current working directory as their base path. You can read more in the [RFC](https://github.com/eslint/rfcs/pull/37).
 
+### Plugins loaded from config file directory
+
+ESLint has long had a problem loading plugins from shared configs due to the way plugin loading was handled. This problem required using peer dependencies and a post-install script to allow shared configs to reference plugins. In v7.0.0, plugins will be loaded relative to the configs that reference them, allowing shared configs to include plugins as dependencies directly. You can read more in the [RFC](https://github.com/eslint/rfcs/pull/47).
```

Comment:
> allowing shared configs to include plugins as dependencies directly.

No.
RFC47 doesn't intend to allow shared configs to include plugins as dependencies directly. See detail: https://github.com/eslint/rfcs/blob/master/designs/2019-plugin-loading-improvement/README.md#the-plugin-uniqueness-with-shareable-configs

Reply:
Oh thanks. That RFC was a bit hard to understand for me, so I missed this part.